cmd/wallets.go

```diff
@@ -52,6 +53,12 @@ func init() {
 	must(viper.BindPFlag("wallets-in-migration-flag", walletsCmd.PersistentFlags().Lookup("wallets-in-migration-flag")))
 	must(viper.BindEnv("wallets-in-migration-flag", "WALLETS_IN_MIGRATION"))
 
+	// ENABLE_LINKING_DRAINING - enable ability to link wallets and drain wallets
+	walletsCmd.PersistentFlags().BoolVarP(&enableLinkingDrainingFlag, "enable-linking-draining-flag", "", false,
+		"the in-migration flag disabling the wallets link feature")
```

Comment:
small nit - may want to tweak this, seems the same as above

Reply:
https://github.com/brave-intl/bat-go/pull/469/commits/2a56be2b9f36338aab261e722af4d8e3f9a4c7dc